.ci/azure-pipelines/azure-pipelines.yaml

```diff
@@ -126,25 +130,28 @@ stages:
     displayName: Build MSVC
     dependsOn: formatting
     jobs:
-      - job: vs2017
-        displayName: Windows VS2017 Build
+      - job: Windows
+        displayName: Windows Build
         pool:
-          vmImage: 'vs2017-win2016'
+          vmImage: 'windows-2019'
         strategy:
           matrix:
             x86:
+              CONTAINER: winx86
               PLATFORM: 'x86'
               ARCHITECTURE: 'x86'
-              GENERATOR: 'Visual Studio 15 2017'
+              GENERATOR: '"Visual Studio 16 2019" -A Win32'
```

Comment:
Why are we moving from Visual Studio 15? Is it EOL?

Reply:
No, I'm just used to use VS16, but maybe we should have the x86 run VS15 and x64 run VS16?

Reply:
Either is fine, TBH. It's just x86 is consistently failing since a few PRs (even this)

Reply:
Can see it makes a segfault. We would have to look into  that.

Reply:
This error is fixed now.